src/main/java/edu/hawaii/its/groupings/access/User.java

```diff
@@ -16,6 +19,17 @@ public User(String uid, String uhUuid, Collection<GrantedAuthority> authorities)
         setUhUuid(uhUuid);
     }
 
+    public User(String uid, String uhUuid, Collection<GrantedAuthority> authorities,
+            String givenName, String cn, String email) {
+        super(uid, "", authorities);
+        setUhUuid(uhUuid);
+        Map<String, Object> defaultAttributes = new HashMap<>();
+        defaultAttributes.put("cn", Collections.singletonList(cn));
+        defaultAttributes.put("mail", Collections.singletonList(email));
+        defaultAttributes.put("givenName", Collections.singletonList(givenName));
+        this.setAttributes(new UhAttributes(defaultAttributes));
```

Comment:
Since the rest of the class does not use the `this` keyword, remove the `this.` from `this.setAttributes` to make it consistent.